### PR TITLE
[blog] Launch of Segragated Witness Testnet

### PR DESCRIPTION
**Pull request by btcdrak**
Todos

 - [ ] Depends on #55*

* Travis tests will fail until #55 is merged

### CODE REVIEW

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Bitcoin Core Ecosystem
+
+There is broad excitement and anticipation as far as what providers and other exchange operators will create with the fundamental developments and innovations included in this release. So far, the [most popular wallets][segwit_adoption] and supporting libraries have stated they will support segwit including Ledger, Trezor, Electrum, and Bitgo. Additionally, work on numerous other libraries such as bitcoinj, bitcoinjs, pycoin and bitcore has already begun.
+
+A faucet is available for "segnet" coins [here](https://segwit.greenaddress.it/faucet/).
+
+Early previews of third party wallet support are available at:
+
+- [mSIGNA](https://github.com/ciphrex/mSIGNA/tree/segwit3) (wallet source-doce)
```
Comment (CodeShark) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: source-doce?

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+If you are a wallet user interested in following the status of your wallet’s implementation, please [follow their progress][segwit_adoption] which will be regularly updated. The testing period will proceed according to the Bitcoin Core roadmap. If you would also like to list your wallet or library support, you can do so [here](https://github.com/bitcoin-core/website/pull/30).
+
+If you are a wallet provider, and interested in instructions for how to upgrade to a segwit-compatible wallet, please read the [migration guide](/en/segwit_wallet_dev)
+
+## Testing
+
+Finally and most importantly, please help testing the Segwit Testnet!
+
+Once compiled from the segwit branch, add `-segnet` to the standard `bitcoind` and `bitcoin-cli` command line.
```
Comment (dexX7) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: It's not clear to which "segwit branch" you refer here. Might be considered to put a link into the "early preview" section.

File: _posts/en/posts/2016-01-21-segwit-testnet.md (hunk 1 above)
Comment (btcdrak) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: fixed

File: _posts/en/posts/2016-01-21-segwit-testnet.md (hunk 2 above)
Comment (btcdrak) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: I made some adjustments.

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Bitcoin Core Ecosystem
+
+There is broad excitement and anticipation as far as what providers and other exchange operators will create with the fundamental developments and innovations included in this release. So far, the [most popular wallets][segwit_adoption] and supporting libraries have stated they will support segwit including Ledger, Trezor, Electrum, and Bitgo. Additionally, work on numerous other libraries such as bitcoinj, bitcoinjs, pycoin and bitcore has already begun.
+
+A faucet is available for "segnet" coins [here](https://segwit.greenaddress.it/faucet/).
+
+Early previews of third party wallet support are available at:
+
+- [mSIGNA](https://github.com/ciphrex/mSIGNA/tree/segwit3) (wallet source-code)
```
Comment (CodeShark) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: New URL: https://github.com/ciphrex/mSIGNA/tree/segwit

Comment (btcdrak) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: fixed

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+If you are a wallet user interested in following the status of your wallet’s implementation, please [follow their progress][segwit_adoption] which will be regularly updated. The testing period will proceed according to the Bitcoin Core roadmap. If you would also like to list your wallet or library support, you can do so [here](https://github.com/bitcoin-core/website/pull/30).
+
+If you are a wallet provider, and interested in instructions for how to upgrade to a segwit-compatible wallet, please read the [migration guide](/en/segwit_wallet_dev)
+
+There is also an IRC channel `segwit-dev` available on irc.freenode.net.
+
+## Testing
+
+Finally and most importantly, please help testing the Segwit Testnet!
```
Comment (instagibbs) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: s/testing/test/

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+The source-code can be located [here](https://github.com/sipa/bitcoin/tree/segwit) checkout the `segwit` branch.
```
Comment (instagibbs) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: I know you know, but segwit3 branch

File: _posts/en/posts/2016-01-21-segwit-testnet.md (hunk 4 above)
Comment (btcdrak) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: Fixed

**Comment by btcdrak** [comment]
This PR requires some segwit people to ACK it for accuracy @CodeShark @sipa @cfields @morcos @sdaftuar

**Comment by kanzure** [comment]
@djat authorship attribution is missing (along with other authors); if we are not writing down authorship attribution, then OK.

I have looked at this content for a few days now and can ACK from my perspective, but not for segwit detail accuracy, I'll leave that up to the others.......

**Comment by jl2012** [comment]
ACK

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+- [mSIGNA](https://github.com/ciphrex/mSIGNA/tree/segwit) (wallet source-code)
+- [Green Address](https://segwit.greenaddress.it/) (web wallet)
+
+## How to get involved
+
+If you are a wallet user interested in following the status of your wallet’s implementation, please [follow their progress][segwit_adoption] which will be regularly updated. The testing period will proceed according to the Bitcoin Core roadmap. If you would also like to list your wallet or library support, you can do so [here](https://github.com/bitcoin-core/website/pull/30).
+
+If you are a wallet provider, and interested in instructions for how to upgrade to a segwit-compatible wallet, please read the [migration guide](/en/segwit_wallet_dev)
+
+There is also an IRC channel `segwit-dev` available on irc.freenode.net.
```
Comment (kanzure) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: We should use the bitcoin-dev IRC channel for segwit discussion..... see http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2016-January/012256.html

**Comment by CodeShark** [comment]
ACK

**Comment by instagibbs** [comment]
ACK

**Comment by btcdrak** [comment]
Got a verbal ACK from @sipa on IRC.

**Comment by btcdrak** [comment]
@kanzure I asked djat and he said no attribution necessary, prefers without.

File: _posts/en/posts/2016-01-21-segwit-testnet.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+This significant innovation leads to dramatically more efficient use of the Bitcoin blockchain, while simultaneously opening up exciting new possibilities and opportunities for the broader Bitcoin ecosystem, particularly around smart contract applications and dramatically faster transactions, while providing the groundwork for more advanced features and possibilities in the future.
+
+This development began with a research effort by Bitcoin Core developer, Dr. Pieter Wuille, initially focused on addressing transaction malleability, a longstanding and well-known concern and priority. However, in the process of this research, and in the narrowing toward a solution, additional properties of the solution were discovered that allow for increasing the block size while also simultaneously opening up some incredibly exciting secondary benefits.
+
+This effort was initiated by Dr. Pieter Wuille, but included contributions from many others, with particular thanks to Eric Lombrozo, Johnson Lau, Alex Morcos, Nicolas Dorier, Bryan Bishop, Gregory Maxwell, Peter Todd, Cory Fields, Sudhas Daftuar, and Luke-Jr.
```
Comment (sdaftuar) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: "Suhas"

Comment (btcdrak) [inline _posts/en/posts/2016-01-21-segwit-testnet.md]: fixed!

**Pull request merged**